roles/validate/files/rules/required_rules/501_policy_cross_reference.py

```diff
@@ -0,0 +1,80 @@
+class Rule:
+    id = "501"
+    description = "Verify Policy Cross Reference Between Policies, Groups, and Switches"
+    severity = "HIGH"
+
+    @classmethod
+    def match(cls, inventory):
+        policies = []
+        groups = []
+        topology_switches = []
+        results = []
+
+        if inventory.get("vxlan", None):
+            if inventory["vxlan"].get("policy", None):
+                if inventory["vxlan"].get("policy").get("policies", None):
+                    policies = inventory["vxlan"]["policy"]["policies"]
+                    for policy in policies:
+                        filename = policy.get("filename", None)
+
+                        if ((filename and policy.get("template_name", None) and policy.get("template_vars", None)) or
+                                (filename and policy.get("template_vars", None))):
+                            results.append(
+                                "Policy definitions are filename with .config or .cfg that defaults template_name to NDFC freeform or "
```

Comment:
This result message is a bit confusing to me in the way it reads.

Reply:
Reworded result msg.